nexus_demos/config/workcell_1_bts/invalid_place_on_conveyor.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version='1.0'?>
```

Comment:
Could we either reduce the number of steps in the BT (even just have one bad step) or call out which step is invalid via coments?

Reply:
https://github.com/osrf/nexus/pull/84/commits/7ab4f2572e3c7da6d19f535ecd5baa922ad248b6